texttospeech/docs/index.rst

```diff
@@ -9,8 +9,8 @@ models.
 
 .. _Alpha: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/master/README.rst
 .. _Cloud Text-to-Speech API: https://cloud.google.com/texttospeech
-.. _Client Library Documentation: https://googlecloudplatform.github.io/google-cloud-python/stable/texttospeech/api.html
-.. _Product Documentation: https://cloud.google.com/texttospeech
+.. _Client Library Documentation: https://googlecloudplatform.github.io/google-cloud-python/stable/texttospeech/usage.html
```

Comment:
Not a valid link (nor is the original, because no `stable`, but `api.html` is the correct document under `latest`).